_posts/2023-12-14-cockpit-307.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 307
+author: jelle
+date: '2023-12-14'
+tags: cockpit, machines
+slug: cockpit-307
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 307:
+
+
+## Storage redesign
+
+The Storage page has been redesigned with the goal of making more things visible more of the time. The overview page now shows all storage objects in a table and allows almost all operations to be performed on them right from the table.
```

Comment:
"more of the time" sounds weird. Perhaps "in all views" or just drop it?

"to be" → "can be"